docs/reference/configuration/queue.rst

```diff
@@ -0,0 +1,479 @@
+.. _queue-system-chapter:
+
+Queue system
+============
+
+The queue system is responsible for scheduling, running, monitoring and acting
+on a set of realizations.
+
+A number of queue systems are pre-installed with ERT:
+
+- ``LOCAL`` — run all computations locally, on your machine. :ref:`Details.<local-queue>`
+- ``LSF`` — send computation to an LSF cluster. :ref:`Details.<lsf-systems>`
```

Comment:
"run all" vs "send" ?

Reply:
I guess I was trying to convey the idea that LOCAL does things nearby, and the other 'send' computation somewhere else.

But you think it would be clearer to just use 'run all' for all of them? Probably true...

Reply:
Hmm, no I think you might be right in your initial suggestion.